stream_alert_cli/package.py

```diff
@@ -193,25 +205,27 @@ def _upload(self, package_path):
             [boolean] Indicating a successful upload
 
         Raises:
-            Exception if client put_object fails
+            Exception if the Lambda package put object fails to S3
```

Comment:
This no longer raises an exception and only returns a boolean now - maybe either update the docstring or still raise the exception below.